plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -515,6 +518,17 @@ private String createContainer(String workspaceId,
                                            .collect(toMap(Function.identity(),
                                                           value -> new PortBinding[0])))
                   .withVolumesFrom(toArrayIfNotNull(service.getVolumesFrom()));
+        
+        if (volumesFrom != null) {
```

Comment:
The code has been added to addSystemWideContainerSettings.  I'm not sure what to suggest to make the test framework less confusing, perhaps after I spend more time with it I'll get some ideas. :)